### PR TITLE
Prepare for releasing nydus-api v0.1.0

### PR DESCRIPTION
**Pull request by jiangliu**
Prepare for releasing nydus-api v0.1.0

### CODE REVIEW

File: api/LICENSE-BSD-3-Clause [hunk 1]
```diff
@@ -0,0 +1 @@
+../LICENSE-BSD-3-Clause
```
Comment (bergwolf) [inline api/LICENSE-BSD-3-Clause:1]: Is the license file required for each crate?

File: api/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# nydus-error
```
Comment (bergwolf) [inline api/README.md:1]: nydus-api

File: api/README.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+The `nydus-api` crate defines the Nydus Image Service APIs and related data structures.
+
+## Support
+
+**Platforms**:
+- x86_64
+- aarch64
+
+**Operating Systems**:
+- Linux
```
Comment (bergwolf) [inline api/README.md:12]: - Darwin

File: api/src/http.rs [hunk 4]
```diff
@@ -398,32 +590,34 @@ lazy_static! {
...
 
-        // Global
+        // Common
         r.routes.insert(endpoint_v1!("/daemon"), Box::new(InfoHandler{}));
-        r.routes.insert(endpoint_v2!("/daemon"), Box::new(InfoHandler{}));
-        r.routes.insert(endpoint_v1!("/daemon/backend"), Box::new(FsBackendInfo{}));
-        r.routes.insert(endpoint_v2!("/daemon/backend"), Box::new(FsBackendInfo{}));
+        r.routes.insert(endpoint_v2!("/daemon"), Box::new(InfoV2Handler{}));
         r.routes.insert(endpoint_v1!("/daemon/events"), Box::new(EventsHandler{}));
         r.routes.insert(endpoint_v2!("/daemon/events"), Box::new(EventsHandler{}));
```
Comment (bergwolf) [inline api/src/http.rs:597]: Can we remove these duplicated APIs from v2? If anyone wants to use them, they should just use the v1 APIs.

File: api/LICENSE-BSD-3-Clause (hunk 1 above)
Comment (jiangliu) [reply api/LICENSE-BSD-3-Clause:1]: It's

File: api/README.md (hunk 2 above)
Comment (jiangliu) [reply api/README.md:1]: Fixed

File: src/bin/nydusd/blob_cache.rs [hunk 5]
```diff
@@ -320,9 +320,10 @@ impl BlobCacheMgr {
             ));
         }
 
-        let prefetch_config = match serde_json::from_value(entry.fs_prefetch.clone()) {
-            Ok(fs_prefetch) => fs_prefetch,
-            Err(_e) => Default::default(),
+        let prefetch_config = if entry.fs_prefetch.is_some() {
```
Comment (liubogithub) [inline src/bin/nydusd/blob_cache.rs:323]: This change seems irrelevant to the patch.
